src/core/scene/a-scene.js

```diff
@@ -113,6 +115,12 @@ module.exports.AScene = registerElement('a-scene', {
         // Handler to exit VR (e.g., Oculus Browser back button).
         this.onVRPresentChangeBound = bind(this.onVRPresentChange, this);
         window.addEventListener('vrdisplaypresentchange', this.onVRPresentChangeBound);
+
+        // Enter VR on `vrdisplayactivate` (e.g. putting on Rift headset).
```

Comment:
This logic will go into the `link` will be on the link system but we can keep it for now